Sources/Kitura/bodyParser/BodyParser.swift

```diff
@@ -115,9 +117,9 @@ public class BodyParser: RouterMiddleware {
 
     /// Read incoming message for Parse
     ///
-    /// - Parameter message: message coming from the socket
+    /// - Parameter message: Message coming from the socket
     /// - Parameter parser: ((NSData) -> ParsedBody?) store at parserMap
-    /// - Returns: the parsed body
+    /// - Returns: The parsed body
```

Comment:
Why are there no usage examples for the init, parse, readBodyData and handle calls?